src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -150,7 +174,7 @@ namespace AppInstaller::CLI::Execution
         Args Args;
 
         // Creates a copy of this context as it was at construction.
-        Context Clone();
+        virtual std::unique_ptr<Context> Clone();

```

Comment:
>virtual std::unique_ptr<Context> Clone(); [](start = 8, length = 41)

Why does this need to be a unique_ptr?  I agree that I didn't put in all the copy and move functions, but that would be the better choice.

Reply:
Should have seen the virtual, got it now.

---
In reply to: [503507999](https://github.com/microsoft/winget-cli/pull/588#discussion_r503507999) [](ancestors = 503507999)